app/build.gradle

```diff
@@ -260,9 +260,9 @@ dependencies {
     implementation project(':element-hiding-store')
 
     implementation project(':sync-api')
-    internalImplementation project(':sync-impl')
-    internalImplementation project(':sync-store')
-    internalImplementation project(':sync-lib')
+    implementation project(':sync-impl')
```

Comment:
I know we discussed this, but leaving it as a comment.

This will include :sync modules for all flavors (that includes our crypto library). I'm not confident this is not going to be an issue when publishing in other stores (we discussed the reasons, our friend the binary). That being said, I'm ok if you want to push for it and taking care of any issues on those stores.